translations/messages+intl-icu.de.yml

```
@@ -2764,6 +2764,7 @@ public.index.filter.gfk.location.results: |
         =0 {Es wurden zu der Kommune folgende Verfahren gefunden:}
         =1 {Es wurden zu der Kommune {locationName} folgende Verfahren gefunden:}
     }
+public.index.heading: "DiPlanBeteiligung: Plattform zur öffentlichen Mitwirkung bei räumlichen Planungen und Genehmigungsverfahren"
```

As this is project specific, it would be better placed as an override. Or better, you could pass the project name as variable:

```suggestion
public.index.heading: "{projectName}: Plattform zur öffentlichen Mitwirkung bei räumlichen Planungen und Genehmigungsverfahren"
```